docs/prod-install.md

```diff
@@ -87,16 +87,6 @@ These settings include:
   correctly, since outgoing email is the most common configuration
   problem.
 
-- `AUTHENTICATION_BACKENDS`: a list of enabled authentication
```

Comment:
Maybe leave a brief ad here, e.g.

"If desired, you can also [configure additional authentication backends]() while you're editing `/etc/zulip/settings.py`.  Note, however, that the default email backend is the only one that supports creating a new organization."